tests/integration/targets/vmware_rest_appliance-multi-hosts/tasks/appliance_ntp.yml

```diff
@@ -0,0 +1,65 @@
+---
+- name: Manage NTP Configuration on vCenter
+  block:
+    - name: Reset the NTP configuration
```

Comment:
Should NTP configuration be restored at the end of the test?

Reply:
I thought about that, but I don't recall the primary NTP that was configured before this test lol
But anyway, it was probably something like "time.vmware.com", and the last one to be configured is "time.google.com", so I think google is better?